2-mandatory/1-create-functions.js

```diff
@@ -33,9 +36,13 @@ Write a function that:
 - Returns a new array containing the same elements, but without the element at the passed index.
 */
 
-function remove() {
+function remove(array, index) {
+  let newArray = [...array]
+  newArray.splice(index, 1)
+  return newArray
```

Comment:
`.splice()` mutate the original array, so really good that you're creating a new array here